### PR TITLE
Add MaxAckDelay sub-section

### PR DESCRIPTION
**Pull request by ianswett**

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -206,6 +206,17 @@ Min RTT is the minimum RTT measured over the connection, prior to adjusting by
...
 underestimation of min RTT, which in turn prevents underestimating smoothed RTT.
 
+### Maximum Ack Delay
+
+QUIC is able to explicitly model delay at the receiver via the ack delay
+field in the ACK frame.  Therefore, QUIC diverges from TCP by calculating a
+MaxAckDelay dynamically, instead of assuming a constant delayed ack timeout
+for all connections.
+
+MaxAckDelay is the maximum ack delay supplied in an incoming ACK frame.
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md]: In all incoming ACK frames?

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -206,6 +206,17 @@ Min RTT is the minimum RTT measured over the connection, prior to adjusting by
...
+MaxAckDelay excludes ack delays that aren't included in an RTT sample because
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md]: Should we say "that don't have a larger Largest Acknowledged"?

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -338,15 +351,10 @@ conditions:
...
-MaxAckDelay is the maximum ack delay supplied in an incoming ACK frame.
-MaxAckDelay excludes ack delays that aren't included in an RTT sample because
-they're too large and excludes those which reference an ack-only packet.
-
-QUIC diverges from TCP by calculating MaxAckDelay dynamically, instead of
-assuming a constant delayed ack timeout for all connections.  QUIC includes this
-in all probe timeouts, because it assume the ack delay may come into play,
-regardless of the number of packets outstanding.  TCP's TLP assumes if at least
-2 packets are outstanding, acks will not be delayed.
+QUIC includes MaxAckDelay in all probe timeouts, because it assume the ack
```
Comment (rpaulo) [inline draft-ietf-quic-recovery.md]: "assumes"

**Comment by ianswett** [comment]
Any other comments?  This is purely editorial, so it'd be nice to get it in soon before it rots.

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -206,6 +206,19 @@ Min RTT is the minimum RTT measured over the connection, prior to adjusting by
...
+QUIC is able to explicitly model delay at the receiver via the ack delay
+field in the ACK frame.  Therefore, QUIC diverges from TCP by calculating a
+MaxAckDelay dynamically, instead of assuming a constant delayed ack timeout
+for all connections.
+
+MaxAckDelay is the maximum ack delay supplied in an all incoming ACK frames.
+MaxAckDelay excludes ack delays that aren't included in an RTT sample because
+they're too large or the largest acked has already been acknowledged.
+MaxAckDelay also excludes ack delays where the largest ack references an
+ACK-only packet.
```
Comment (siyengar) [inline draft-ietf-quic-recovery.md]: Lgtm

File: draft-ietf-quic-recovery.md [hunk 5]
```diff
@@ -206,6 +206,19 @@ Min RTT is the minimum RTT measured over the connection, prior to adjusting by
...
+
+QUIC is able to explicitly model delay at the receiver via the ack delay
+field in the ACK frame.  Therefore, QUIC diverges from TCP by calculating a
+MaxAckDelay dynamically, instead of assuming a constant delayed ack timeout
+for all connections.
+
+MaxAckDelay is the maximum ack delay supplied in an all incoming ACK frames.
+MaxAckDelay excludes ack delays that aren't included in an RTT sample because
+they're too large or the largest acked has already been acknowledged.
+MaxAckDelay also excludes ack delays where the largest ack references an
```
Comment (MikeBishop) [inline draft-ietf-quic-recovery.md]: You've used "largest ack" and "largest acked" here.  Elsewhere in the document, you also use "largest acknowledged."  Might be worth picking one and using it consistently?

**Pull request merged**